connect/runtime/src/test/java/org/apache/kafka/connect/runtime/distributed/WorkerCoordinatorTest.java

```diff
@@ -110,32 +113,57 @@ public void setup() {
                 rebalanceListener);
 
         configState1 = new ClusterConfigState(
-                1L, Collections.singletonMap(connectorId, 1),
-                Collections.singletonMap(connectorId, (Map<String, String>) new HashMap<String, String>()),
-                Collections.singletonMap(connectorId, TargetState.STARTED),
-                Collections.singletonMap(taskId0, (Map<String, String>) new HashMap<String, String>()),
+                1L, Collections.singletonMap(connectorId1, 1),
```

Comment:
This first argument (`1L`) can be missed, given that that's the only line in which we pass 2 arguments. I'd add one argument per line.